codegenerator/cli/templates/static/codegen/src/Address.res

```diff
@@ -1 +1,22 @@
-type t = Ethers.ethAddress
\ No newline at end of file
+@genType.import(("./OpaqueTypes.ts", "Address"))
+type t
+
+let schema =
+  S.string->S.setName("Address")->(Utils.magic: S.t<string> => S.t<t>)
+
+external toString: t => string = "%identity"
+
+external unsafeFromString: string => t = "%identity"
+
+module Evm = {
+  @module("ethers") @scope("ethers")
+  external fromStringOrThrow: string => t = "getAddress"
```

Comment:
Lets rather use viem for this. It's a much faster implementtion

Reply:
I'll do in another PR